Modules/mathmodule.c

```diff
@@ -2348,6 +2348,13 @@ math_fmod_impl(PyObject *module, double x, double y)
         return PyFloat_FromDouble(x);
     errno = 0;
     r = fmod(x, y);
+    /* Some platforms (e.g. Windows 10 with MSC v.1916) loose sign
```

Comment:
Can you surround this code with `#ifdef _MSC_VER`? It seems like only Windows needs this workaround so far.